docs/framework.md

```diff
@@ -3114,20 +2361,9 @@ Date of variation
 
 State the actual annual measured levels of demand or stated levels of demand in the providers report or contract managers report. Use the following format
 
- Structured data about estimated demand should be provided in the `contract/implementation/metrics` section of an OCDS release, using an array of metrics building blocks.
-
-A metric with the `id` 'demand' should be given, with a series of actual `observations` that capture the actual demand for a given period.
-
-These estimates can be disaggregated by any number of dimensions contained as simple fields within the `observation/dimensions` object. 
-
-**Schema**: Information can be provided using the following OCDS fields.
+Structured data about estimated demand should be provided using [Metric](https://extensions.open-contracting.org/en/extensions/metrics/master/schema/#metric) objects in the [`contracts/implementation/metrics`](../reference/schema/#release-schema.json,,contracts/0/implementation/metrics) array.
 
-```eval_rst
-.. jsonschema:: _static/patched/release-schema.json
-    :include: contracts/0/implementation/metrics
-    :collapse: 
-    :nocrossref:
-```
+A metric with an `id` of 'demand' should be given, with a series of [`observations`](../reference/schema/#release-schema.json,,contracts/0/implementation/metrics/0/observations) that capture the actual demand for a given period. These estimates can be disaggregated by any number of dimensions contained as simple fields within each observation's [`dimensions`](../reference/schema/#release-schema.json,,contracts/0/implementation/metrics/0/observations/0/dimensions) object. 
```

Comment:
Clarify the meaning of 'simple fields'. Suggest applying to all other occurrences too.

```suggestion
A metric with an `id` of 'demand' should be given, with a series of [`observations`](../reference/schema/#release-schema.json,,contracts/0/implementation/metrics/0/observations) that capture the actual demand for a given period. These estimates can be disaggregated by any number of dimensions contained as key-value pairs within each observation's [`dimensions`](../reference/schema/#release-schema.json,,contracts/0/implementation/metrics/0/observations/0/dimensions) object. 
```

Reply:
This was present in the original, but happy to make the change.

Reply:
Fixed 0f8979b